<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>535</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blink</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/blink48.png</normaloff>icons/blink48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="AccountSelector" name="identity">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SearchBox" name="search_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="inactiveText" stdset="0">
       <string>Search Contacts or Enter Address</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="identity_layout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="image">
        <property name="minimumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>icons/default-avatar.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="name_note_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="name_status_layout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="LineEdit" name="display_name">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string>background-color: transparent;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="inactiveText" stdset="0">
             <string>Type your name here</string>
            </property>
            <property name="widgetSpacing" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="status">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>18</height>
             </size>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Available</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/green-dot.svg</normaloff>icons/green-dot.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Away</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/yellow-dot.svg</normaloff>icons/yellow-dot.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Busy</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/red-dot.svg</normaloff>icons/red-dot.svg</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>On the phone</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/red-dot.svg</normaloff>icons/red-dot.svg</iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="LineEdit" name="activity_note">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>background-color: transparent;</string>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="inactiveText" stdset="0">
           <string>Add an activity note here</string>
          </property>
          <property name="widgetSpacing" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="switch_view_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="text">
       <string>Sessions</string>
      </property>
      <property name="iconSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="main_view">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="contacts_panel">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="contacts_view">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="contact_list_panel">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="ContactListView" name="contact_list">
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DragDrop</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="search_panel">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="search_topbox_layout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="search_topbox_spacer_1">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="search_topbuttons_layout">
                <property name="spacing">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QPushButton" name="back_to_contacts_button">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::TabFocus</enum>
                  </property>
                  <property name="text">
                   <string>Back to contacts</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="add_search_contact_button">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::TabFocus</enum>
                  </property>
                  <property name="text">
                   <string>Add contact</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="search_topbox_spacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="search_view">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="search_list_panel">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="ContactSearchListView" name="search_list">
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                  <property name="dragDropMode">
                   <enum>QAbstractItemView::DragDrop</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::ExtendedSelection</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="not_found_panel">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="not_found_label">
                  <property name="text">
                   <string>No matching contacts found.
Press Return to start a call to
the typed address, or use the
buttons below.</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="contact_buttonbox_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QToolButton" name="add_contact_button">
            <property name="minimumSize">
             <size>
              <width>29</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/plus18.svg</normaloff>icons/plus18.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="button_box_spacer_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="dial_buttons_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="audio_call_button">
              <property name="minimumSize">
               <size>
                <width>29</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>29</width>
                <height>24</height>
               </size>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/audio.png</normaloff>icons/audio.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ToolButton" name="im_session_button">
              <property name="minimumSize">
               <size>
                <width>29</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>29</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string>IM</string>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ToolButton" name="ds_session_button">
              <property name="minimumSize">
               <size>
                <width>29</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>29</width>
                <height>24</height>
               </size>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>icons/display18.png</normaloff>icons/display18.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>18</width>
                <height>18</height>
               </size>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="button_box_spacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="silent_button">
            <property name="minimumSize">
             <size>
              <width>29</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>M</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/bell-on.png</normaloff>
              <normalon>icons/bell-off.png</normalon>icons/bell-on.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sessions_panel">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListView" name="session_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="session_buttons_layout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="hangup_all_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Hangup all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="conference_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="text">
             <string>Conference</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="session_buttons_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="mute_button">
            <property name="minimumSize">
             <size>
              <width>29</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>29</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/mic-on.png</normaloff>
              <normalon>icons/mic-off.png</normalon>icons/mic-on.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBlink">
    <property name="title">
     <string>Blink</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuBlink"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchBox</class>
   <extends>QLineEdit</extends>
   <header>blink.widgets.searchbox</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>blink.widgets.lineedit</header>
  </customwidget>
  <customwidget>
   <class>ContactListView</class>
   <extends>QListView</extends>
   <header>blink.contacts</header>
  </customwidget>
  <customwidget>
   <class>ContactSearchListView</class>
   <extends>QListView</extends>
   <header>blink.contacts</header>
  </customwidget>
  <customwidget>
   <class>AccountSelector</class>
   <extends>QComboBox</extends>
   <header>blink.accounts</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>blink.widgets.buttons</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>search_box</tabstop>
  <tabstop>identity</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>main_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
